### PR TITLE
Publicize query

### PR DESCRIPTION
**Pull request by MarquiseRosier**
Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues
https://github.com/adobe/helix-run-query/issues/53

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -56,16 +42,35 @@ async function main(params) {
...
     googlebigquery: 'https://www.googleapis.com/discovery/v1/apis/bigquery/v2/rest',
-  }),
-};
+  })(params);
+}
+
+async function main(params) {
+  if (params.__ow_headers && ('x-token' in params.__ow_headers) && ('x-service' in params.__ow_headers)) {
+    // eslint-disable-next-line no-param-reassign
+    params.token = params.__ow_headers['x-token'];
```
Comment (tripodsan) [inline src/index.js]: what is the `token` here? will this come from the package params?

File: src/index.js [hunk 2]
```diff
@@ -56,16 +42,35 @@ async function main(params) {
...
+  let action = runExec;
+  action = openWhiskWrapper(action, {
     token_param: 'EPSAGON_TOKEN',
     appName: 'Helix Services',
-    metadataOnly: false,
-    ignoredKeys: [/^[A-Z0-9_]+$/, 'token'],
-  }),
-  {
+    metadataOnly: false, // Optional, send more trace data
+    ignoredKeys: ['token', 'GOOGLE_PRIVATE_KEY', /[A-Z0-9_]+/],
```
Comment (tripodsan) [inline src/index.js]: `GOOGLE_PRIVATE_KEY` is already covered by the regexp.

File: src/index.js (hunk 1 above)
Comment (MarquiseRosier) [reply src/index.js]: @tripodsan they will come from the request headers when someone sends a GET request for a query that requires authorization

File: src/index.js (hunk 2 above)
Comment (MarquiseRosier) [reply src/index.js]: Whoops great catch!

**Pull request merged**